packages/components/src/query-controls/README.md

```diff
@@ -123,17 +123,16 @@ An array of author IDs that is passed into an `AuthorSelect` sub-component.
 -   Required: No
 -   Platform: Web
 
-#### selectedAuthorId
+#### `selectedAuthorId`: `string`
```

Comment:
This is actually passed as a string:

<img width="966" alt="Screen Shot 2022-12-21 at 4 29 52 PM" src="https://user-images.githubusercontent.com/4063887/209031698-0308f076-95b8-4f47-bf43-cf35900aa9e8.png">